### ISSUE TITLE
What is the threshold of the logits?

### ISSUE THREAD

**Issue by yizhang-unifr** [baeseongsu/ehrxqa#8]
Hi, I have a question.
Could you kindly provide the threshold value of the logits used in the experiments for the MimicCXR VQA task as mentioned in your paper?

Thank you for sharing your works!

**Comment by baeseongsu** [comment]
Hi, @yizhang-unifr!

Thank you for asking about this information! The threshold value of 0.5 is used in our experiment. You can check our code repository for evidence: [MIMIC-CXR-VQA experiment](https://github.com/baeseongsu/mimic-cxr-vqa/blob/984da408fc5421d8f3718c58859b9048077f190a/mimiccxrvqa/exp/M3AE/m3ae/gadgets/my_metrics.py#L126) and [EHRXQA experiment](https://github.com/baeseongsu/ehrxqa/blob/724bff13a9a2e430ecd54f32e3ef3789bb7fcdb3/experiment/NeuralSQL/executor/visual_module/m3ae_vqa_module.py#L57).

Best,
Seongsu

**Comment by yizhang-unifr** [comment]
All clear! 
정말 감사합니다!
